src/app/pages/vendor-dashboard/vendor-dashboard.component.html

```diff
@@ -1,32 +1,38 @@
-<!-- <ng-container>
-<mat-sidenav-container>
-  <mat-sidenav #sidenav mode="side" opened class="example-sidenav" [fixedInViewport]="options.value.fixed" [fixedTopGap]="options.value.top"
-  [fixedBottomGap]="options.value.bottom">
-    [fixedInViewport]="options.value.fixed"
-  </mat-sidenav>
-</mat-sidenav-container>
-</ng-container> -->
-<p>Vendor Dashboard</p>
-<!--<SearchBarComponent></SearchBarComponent>
-<p>
-  <mat-form-field appearance="outline">
-    <mat-label>Outline form field</mat-label>
-    <input matInput placeholder="Placeholder">
-    <mat-icon matSuffix>sentiment_very_satisfied</mat-icon>
-    <mat-hint>Hint</mat-hint>
-  </mat-form-field>
-</p>-->
 <mat-form-field appearance="outline" style="width:40vw; font-size: 12px" color="warning">
   <mat-label>Search</mat-label>
   <input matInput placeholder="Search">
```

Comment:
Are you able to get the search text to apply to a variable in the component.ts file? If not don't worry about it. I can connect it up real quick when I connect the routes Robby is adding.